index.md

```diff
@@ -1,277 +1,71 @@
 ---
 layout: lesson
 root: .
-lastupdated: Month Day, Year
-contributors: ["First Last", "First Last", "Pls Add Others"]
-maintainers: ["First Last", "First Last"]
-domain: Domain Name
-topic: Topic
-software:
-dataurl:
-status: Under Development
 ---
 
-Data Carpentry workshops are for any researcher who has data they want to analyze, and no prior computational experience is required. 
-This hands-on workshop teaches basic concepts, skills and tools for working more effectively with data.
-
-The focuses of this workshop will be working with genomics data, and data management & analysis for genomics research. We will cover
-metadata organization in spreadsheets, data organization, connecting to and using cloud computing, the command line for sequence quality
-control and bioinformatics workflows, and R for data analysis and visualization. We will not be teaching any particular bioinformatics 
-tools, but the foundational skills that will allow you to conduct any analysis and analyze the output of a genomics pipeline.
+Data Carpentry’s aim is to teach researchers basic concepts, skills, and tools for working
+with data so that they can get more done in less time, and with less pain. This workshop
+teaches data management and analysis for genomics research including: 
+best practices for organization of bioinformatics projects and data, use of command line 
+utilities, and connecting to and using cloud computing. Depending on which workshop
+track you tack, we will also cover either a) data analysis and visualization in the R 
+programming language or b) the use of command line tools to analyze sequence quality and 
+perform variant calling.
 
 > ## Prerequisites
->
-> There are no pre-requisites, and the materials assume no prior knowledge about the tools. Participants should bring their laptops and plan to participate actively. 
+> 
+> This lesson assumes no prior experience with the tools covered in the workshop. 
+> However, learners are expected to have some familiarity with biological concepts,
+> including the structure of DNA, nucleotide abbreviations, and the 
```

Comment:
do people need to know the structure of DNA?

Maybe "learners are expected to have some familiarity with genomics concepts, including what is a genome, and the general concept of sequencing."

Reply:
I was trying to get at the point that learners need to know what "A, T, C, and G" stand for (or at least that they are nucleotide abbreviations) and that "N" is not a nucleotide abbreviation. They do need to know that genomes are varient within populations in order to understand the story behind the wrangling lesson. I'm happy to change and remove the "structure of DNA".

Reply:
Good point. Could it maybe be "some familiarity with genomics concepts, including what is a genome and DNA, and the general concept of sequencing"